### PR TITLE
Indian Truck Detection Overfitting

### PR DESCRIPTION
**Pull request by Akansha-Mulchandani**
# Pull Request for DL-Simplified 💡

## Issue Title : Indian Truck Detection 

- **Info about the related issue (Aim of the project)** : Finetuning the existing model and introducing newmodels for indian truck detection
- **Name:** Akansha Mulchandani
- **GitHub ID:** Akansha-Mulchandani
- **Email ID:** akanshamulchandani429@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)** GSSOC'24

Closes: #issue number that will be closed through this PR 275

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
-->Finetuned the existing model and added Xception & ResNet 50 model to detect IndianTrucks 
## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made
-->the finetuned models now stand at 98.60 %
-->Xception model accuracy is 90.7%
-->ResNet50 model accuracy is 98.60%
## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Update the README file as well with the new implementations made by you @Akansha-Mulchandani

**Comment by Akansha-Mulchandani** [comment]
Updated

**Comment by Akansha-Mulchandani** [comment]
I had updated the pull request a while ago, is there an issue with that
?because I don’t see it merged

**Review by abhisheks008 — APPROVED** [review]
Approved @Akansha-Mulchandani. Looks good to me!
